python/packages/ai/teams/teams_attachment_downloader.py

```diff
@@ -0,0 +1,156 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import List, Optional
+
+import aiohttp
+from botbuilder.core import TurnContext
+from botbuilder.schema import Attachment
+from botframework.connector.auth import AuthenticationConstants, GovernmentConstants
+
+from .input_file import InputFile, InputFileDownloader
+from .teams_adapter import TeamsAdapter
+
+
+@dataclass
+class TeamsAttachmentDownloaderOptions:
```

Comment:
can we move this to a folder `teams_attachment_downloader` and breakout the different classes into their own files so its easier to read and matches the other features.